templates/base.html

```diff
@@ -45,7 +45,9 @@
         <link rel="stylesheet" href="{% static 'css/submission.css' %}" />
         <link rel="stylesheet" href="{% static 'django_colortag.css' %}" />
         <script src="{% static 'js/polyfill-find.js' %}"></script>
-        <script src="{% static 'js/aplus.js' %}"></script>
+        <script src="{% static 'js/aplus.js' %}"
+                filterHelperTrans="{% trans 'Comparison operators >, <, >=, <=, !=, == are available — i.e. >=200. Use dot as decimal seperator — i.e. 2.1. Regex is also available — i.e. /^\\d+$/g' %}"
```

Comment:
Name like `data-filter-tooltip-text` would discribe the purpose a bit better and also would be in more correct format. It is read using `currentScript.data('filter-tooltip-text')` or `currentScript.data('filterTooltipText')` for example (or `currentScript.attr('data-filter-tooltip-text')`.

p.s.
> the name must not contain capital A to Z letters.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*